lib/src/bindings_player_ffi.dart

```diff
@@ -369,37 +369,50 @@ class FlutterSoLoudFfi {
   late final _getRelativePlaySpeed =
       _getRelativePlaySpeedPtr.asFunction<double Function(int)>();
 
-  /// Play already loaded sound identified by [soundHash].
-  ///
-  /// [soundHash] the unique sound hash of a sound
-  /// [volume] 1.0f full volume
-  /// [pan] 0.0f centered
-  /// [paused] 0 not pause
-  ///
-  /// Return the handle of the sound, [SoundHandle.error] if error
-  SoundHandle play(
+  /// Play already loaded sound identified by [hash]
+  ///
+  /// [hash] the unique sound hash of a sound
```

Comment:
[nit] Should probably refer to `[soundHash]` as that's the name of the parameter.

Reply:
yep!